readme.md

```diff
@@ -15,11 +16,12 @@ We’ve built it to work together with your favourite JSON parsers, be it Gson,
 
 ## How to set it up:
 
-1. Add JitPack repository to root project `build.gradle` file:
+1. Add JitPack or MavenCentral repository to root project `build.gradle` file:
 ```groovy
 allprojects {
     repositories {
         ...
+        mavenCentral()
         maven { url 'https://jitpack.io' }
```

Comment:
Do we need this in readme `maven { url 'https://jitpack.io' }`

Reply:
JitPack is also supported, so I would keep it as example. Is it ok with you?